docs/example.rst

```diff
@@ -1,72 +1,80 @@
+.. include:: links.rst
+
 ===================
 Example Walkthrough
 ===================
 
-The ``CuBIDS`` workflow is currently being used in neuroimaging labs at a number of institutions 
-including University of Pennsylvania, Children's Hospital of Philadelphia, the Child Mind Institute, 
-and University of Minnesota's Masonic Institute for the Developing Brain. To demonstrate the utility of
-``CuBIDS``, here we apply the software to a small example dataset that is included `in our Github repo <https://github.com/PennLINC/CuBIDS/tree/main/cubids/testdata/BIDS_Dataset>`_.
-This example dataset does not contain any PHI. 
+The ``CuBIDS`` workflow is currently being used in neuroimaging labs at a number of institutions
+including University of Pennsylvania, Children's Hospital of Philadelphia, the Child Mind Institute,
+and University of Minnesota's Masonic Institute for the Developing Brain.
+To demonstrate the utility of ``CuBIDS``,
+here we apply the software to a small example dataset that is included
+`in our Github repo <https://github.com/PennLINC/CuBIDS/tree/main/cubids/testdata/BIDS_Dataset>`_.
+This example dataset does not contain any PHI.
 
 Following the installation instructions at :doc:`the installation page <installation>`,
 you should have successfully installed ``CuBIDS``, ``DataLad``, and the ``bids-validator`` inside a
-conda environment titled "cubids". In this example, we use validator version ``1.7.2``.
-Using a different version of the validator may result in slightly different validation
-tsv outputs, but the example should still be useful. 
-
-Throughout this example, we use ``DataLad`` for version control. Although ``DataLad``
-is an optional dependency of ``CuBIDS``, we use it here to demonstrate its
-powerful integration with ``CuBIDS`` and the benefits it can provide its users. 
+conda environment titled "cubids".
+In this example, we use validator version ``1.7.2``.
+Using a different version of the validator may result in slightly different validation tsv outputs,
+but the example should still be useful.
 
-Now that we have installed CuBIDS and all necessary dependencies, we are ready to begin the curation 
-process on our example dataset. We create a ``CuBIDS_Test`` directory to function as our working directory 
-and navigate to it as follows:
+Throughout this example, we use ``DataLad`` for version control.
+Although ``DataLad`` is an optional dependency of ``CuBIDS``,
+we use it here to demonstrate its powerful integration with ``CuBIDS`` and the benefits
+it can provide its users.
 
-mkdir $PWD/CuBIDS_Test 
-cd CuBIDS_Test
+Now that we have installed CuBIDS and all necessary dependencies,
+we are ready to begin the curation process on our example dataset.
+We create a ``CuBIDS_Test`` directory to function as our working directory and navigate to it as follows.
 
 .. code-block:: console
 
+    $ mkdir $PWD/CuBIDS_Test
+    $ cd CuBIDS_Test
```

Comment:
does this make CuBIDS_test/CuBIDS_Test?

Reply:
Ah, I must have pasted it twice. Good catch!

Reply:
Fixed now.